docs/Use_cases/Air_quality_monitoring_use_case.md

```diff
@@ -0,0 +1,229 @@
+# Air quality monitoring - use case
+
+The Coiote DM and Azure IoT Hub integration lets you create custom use cases with data visualization. Follow the tutorial below to see how to set up an inspiringly complex use case in just a few steps.
+
+## Prerequisites
+
+- An active Azure subscription.
+- An active Coiote DM account.
+- An active Microsoft Power BI account.
+- An OpenWeatherMap account with a free API token.
+
+## Creating and configuring an Azure IoT hub and storage account
+
+First you need to add a new IoT hub and a storage account in Azure. Here's how to do it:
+
+### Creating an IoT hub
+
+1. In your Azure portal home view, go to **IoT Hub** and select **Add**.
+![Adding an IoT hub](images/hub_add.png "Add iot hub")
```

Comment:
Why the rectangles are yellow? In any other place, they are red. And one more thing - this yellow is not AVS yellow - Is this intentional?

Reply:
Yellow boxes are for places where you need to provide some input, red boxes are for "action" elements that take you somewhere when you click them. The colours are in no way related to your brand colours (otherwise red should be replaced with what? dark grey?)

Reply:
Ok, that's fine. I was just curious about the reason why some of them are red and some others are yellow.